azurerm/internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -1657,13 +1665,13 @@ func expandKubernetesClusterWindowsProfile(input []interface{}) *containerservic
 	}
 
 	config := input[0].(map[string]interface{})
-
-	adminUsername := config["admin_username"].(string)
-	adminPassword := config["admin_password"].(string)
-
 	profile := containerservice.ManagedClusterWindowsProfile{
-		AdminUsername: &adminUsername,
-		AdminPassword: &adminPassword,
+		AdminUsername: utils.String(config["admin_username"].(string)),
+		AdminPassword: utils.String(config["admin_password"].(string)),
+	}
+
+	if v := config["license"].(string); v != "" {
+		profile.LicenseType = containerservice.LicenseType(v)
 	}
```

Comment:
this means it's not possible to remove this field? can we add a test adding, removing and then adding it again?

Reply:
thanks for pointing it. I have fixed it and add an acctest